data_steward/validation/main.py

```diff
@@ -386,6 +386,55 @@ def generate_metrics(hpo_id, bucket, folder_prefix, summary):
     return report_data
 
 
+def generate_empty_report(hpo_id, bucket, folder_prefix):
+    """
+    Generate an empty report with a "validation failed" error 
+    Also write processed.txt to folder to prevent processing in the future
+
+    :param hpo_id: identifies the HPO site
+    :param bucket: name of the bucket with the submission
+    :param folder_prefix: folder containing the submission
+    :return: report_data: dict whose keys are params in resources/templates/hpo_report.html
+    """
+    report_data = dict()
+    processed_datetime_str = datetime.datetime.now().strftime(
+        '%Y-%m-%dT%H:%M:%S')
+    logging.info(
+        'Folder %s does not follow naming convention %s',
```

Comment:
This is broken.  You must provide a second replacement string if you're going to use this log line.

Reply:
Good catch! Will fix this